development/tools/jobs/kyma/etcd_backup_job_test.go

```diff
@@ -1,4 +1,4 @@
-package tools_test
+package kyma
```

Comment:
each test in `/.../jobs/kyma` directory has package name `kyma_test`, to be consistent it also should have that name

Reply:
changed